packages/wrangler/src/__tests__/pages.test.ts

```diff
@@ -1470,6 +1470,8 @@ describe("pages", () => {
 			expect(std.err).toMatchInlineSnapshot('""');
 		});
 
+		const workerHasD1Shim = (contents: string) => contents.includes("D1_ERROR");
```

Comment:
Not sure why this is abstracted, I only see one reference to it.

Reply:
yeah I totally get it, I felt it could be a bit overkill, but if you look at how it was before:
```ts
// Asserts that the D1 shim has been applied
expect(customWorkerJS).toContain("D1_ERROR");
```
and how it is now:
```ts
const workerHasD1Shim = ...
....
expect(workerHasD1Shim(customWorkerJS as string)).toBeTruthy();
```
I feel like it is so much nicer to have a self-documenting function here instead of restorting to a comment (and without either the code would be a bit unclear I think), as a plus in the future this utility can also be shared among other tests if need be.

So I do think this is better than the alternative, but if you're strongly against it I don't mind reverting this back to how it was 🙂

Reply:
We had a comment plus the check, but felt that a self-documenting function was better. We had planned to re-use elsewhere before realizing that the other place was in a fixture test. But whatever, it's cleaner to have this nice function name which describes the check :)

Reply:
I dont believe in "self-documenting" anything, but I can get behind it if you plan on reusing it.

Reply:
yeah that's a pretty opinionated topic isn't it? 😬

I'm of the general idea that code can't lie but comments can, not to say that I think comments should never be used, but used to a minimum if possible 🙂 (unless we're talking about stuff like tsdoc etc...)

Reply:
With that in mind, even the "self-documenting" function is not that obvious. E.g. why does `"D1_ERROR"` indicate that the shim worked.

Reply:
yeah that's true that we're not explaining the why, but also with the comment we were not explaining the why, so we're conveying the same info as before but without the comment.

in this case indeed I guess it may boil down to personal preference and style of code? :thinking: